bpf/bpf_lxc.c

```diff
@@ -1289,7 +1290,14 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label,
 	}
 
 #ifdef ENABLE_NODEPORT
-	if (ret == CT_NEW || ret == CT_REOPENED) {
+	if (tuple.nexthdr == IPPROTO_TCP) {
+		if (ctx_load_bytes(ctx, l4_off + 12, &tcp_flags, 2) < 0)
```

Comment:
I'm not sure we can use offsetof for the tcp flags since the order of the fields depends on the byte order, plus offsetof can't be used with the bit-field. Any idea?
https://github.com/cilium/cilium/blob/5a143f9d2181037c4e721767ec6e4ee72dbd662f/bpf/include/linux/tcp.h#L24

Reply:
> offsetof can't be used with the bit-field
Oh, yeah you are right. Then, I think we can keep this code as is.